contrib/oauth2/mutt_oauth2.py

```diff
@@ -149,13 +149,16 @@ def writetokenfile():
     if not args.authorize:
         sys.exit('You must run script with "--authorize" at least once.')
     print('Available app and endpoint registrations:', *registrations)
```

Comment:
If the provider is passed as an argument, this print is not useful. Can we integrate this into the input() call for registration?